src/queue_pair.rs

```diff
@@ -43,9 +44,9 @@ pub(crate) static MAX_SEND_WR: u32 = 10;
 /// Maximum value of `recv_wr`
 pub(crate) static MAX_RECV_WR: u32 = 10;
 /// Maximum value of `send_sge`
-pub(crate) static MAX_SEND_SGE: u32 = 10;
+pub(crate) static MAX_SEND_SGE: u32 = 5;
 /// Maximum value of `recv_sge`
-pub(crate) static MAX_RECV_SGE: u32 = 10;
+pub(crate) static MAX_RECV_SGE: u32 = 5;
```

Comment:
Why to change these values?

Reply:
The max_sge is less than 10 for some hardware rdma devices, so I reduced it a little bit to adapt more devices, otherwise, some devices may fail the tests.